third_party/gfootball_engine/CMakeLists.txt

```diff
@@ -19,11 +19,11 @@ project(gameplayfootball)
 set (CMAKE_CXX_STANDARD 14)
 
 if(UNIX)
-   set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -Wall -g -O3 -L/usr/X11/lib -lX11 -lSDL2")
-   set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -Wall -g -O3 -L/usr/X11/lib -lX11 -lSDL2")
+   set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -Wall -g -O3")
```

Comment:
I don't think there is no need to link directly to X11/SDL2 as this taken care of by CMake